packages/nimble-components/src/patterns/dropdown/styles.ts

```diff
@@ -259,5 +267,13 @@ export const styles = css`
                 background-color: rgba(${borderRgbPartialColor}, 0.07);
             }
         `
+    ),
+    themeBehavior(
```

Comment:
Could you try to find a way to write down why we're going with this approach rather than adding a new token? This is just in the interest of traceability if we find another use case that is considering adopting the same pattern. I think the conversation we had this week would be good to write down.

This could be something in the PR description like a short explanation or a link to a discussion, doesn't have to be in code or documentation.